app/assets/stylesheets/blacklight_catalog/_static_pages.scss

```diff
@@ -20,31 +20,34 @@ p.static-blurb-body > a {
 }
 
 .content-row {
-  background: rgba(231,234,241, 0.5);
-/* Light Grey */
+  background: rgba(231, 234, 241, 0.5);
+  /* Light Grey */
 
-border: 1px solid #E7EAF1;
-box-sizing: border-box;
+  border: 1px solid #e7eaf1;
+  box-sizing: border-box;
 }
 
 div.explore-card {
   grid-gap: $navbar-nav-link-padding-x;
   padding: $navbar-padding-x;
   background-color: $mid-blue;
   margin: $facet-label-font-size;
-  -webkit-box-shadow: 0 0 0 1px $explorer-box-shadow, 0 0 0 $facet-label-font-size $mid-blue;
```

Comment:
@CB987 Could you also refactor the code so that any pixel or fraction amount is thrown into a variable?